tests/xfstests/generate_report.pm

```diff
@@ -52,7 +52,7 @@ sub run {
     script_run('curl -O ' . autoinst_url . "/files/status.log; cat status.log > $STATUS_LOG");
 
     # Reload test logs if check missing
-    script_run("if [ ! -d $LOG_DIR ]; then curl -O " . autoinst_url . '/files/opt_logs.tar.gz; tar zxvfP opt_logs.tar.gz; fi');
```

Comment:
you may simplify it with "--create-dirs" param for curl

Reply:
I'm not sure if I understand correctly, here I'm not tent to create folder for downloaded log tar file, just check if necessary to download it.